ghost/i18n/locales/es/portal.json

```diff
@@ -6,171 +6,153 @@
     "{{amount}} off for first {{period}}.": "{{amount}} de descuento por los primeros {{period}}.",
     "{{amount}} off forever.": "{{amount}} de descuento para siempre.",
     "{{discount}}% discount": "{{discount}}% de descuento",
-    "{{memberEmail}} will no longer receive {{newsletterName}} newsletter.": "{{memberEmail}} ya no recibirá boletines de {{newsletterName}}.",
```

Comment:
error discussed in your previous PR.  No accent on the plural.

Reply:
No, I meant boletines no accents, boletín does have accent.

Reply:
Agreed. But you removed the correct bolotines and changed it to boletínes with this PR.

Reply:
Agreed that boletines should not have an accent.  However, your PR removes boletines on line 9 and changes it to boletínes.

Reply:
oh did not notice, sorry to be slow to respond, have been real busy.